postgres_config_test.go

```diff
@@ -23,4 +24,22 @@ var _ = Describe("PostgresConfig", func() {
 			}.ConnectionString()).To(Equal("dbname='atc' host='1.2.3.4' password='password \\\\ with \\' funny ! chars' port=5432 sslmode='verify-full' user='some user'"))
 		})
 	})
+	Describe("Timeout", func() {
```

Comment:
don't think this test is really necessary, there's no behavior actually added or exercised here